install/kubernetes/cilium/charts/nodeinit/templates/daemonset.yaml

```diff
@@ -56,6 +56,18 @@ spec:
                       while crictl ps | grep -v "node-init" | grep -q "POD_cilium"; do sleep 1; done
                     fi
 
+{{- if .Values.azure }}
+                    # Azure specific: Transparent bridge mode is required in
+                    # order for proxy-redirection to work
+                    until [ -f /var/run/azure-vnet.json ]; do
+                      echo waiting for azure-vnet to be created
```

Comment:
@tgraf Why do we do this piece in the `preStop` hook? The way its written sounds like it was intended to be in the main node-init container.

Reply:
This is indeed in the wrong section. I've boostrapped another AKS cluster. It works but relies on restarting Cilium once right now.

Reply:
Will you spin a patch to fix this or would you like me to? I just don't have access to an azure account for testing purposes.

Reply:
PR: https://github.com/cilium/cilium/pull/10383